### PR TITLE
Create Gnoland-TPM-audit.md

### PR DESCRIPTION
**Pull request by Kouteki**
## Description

Publish the audit document created in my 30 days as a TPM.

### CODE REVIEW

File: docs/Discoveries/Gnoland-TPM-audit.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+- Signal is used internally by the Core team when communicating with Manfred and Jae. In rare instances Signal is used to contact Core team members outside working hours.
+- Slack is used internally for more structured communication between Core team members, as well as communication with the rest of the org. There's a plan to phase out Slack in favor of [Matrix](https://matrix.org/), which will allow us to move most Signal conversations to Matrix; no ETA for this at the moment.
+- Notion is used for cross-team project collaboration, and serves as a primary project board for non-technical teams.
+- Email is used rarely, primarily for company wide collaboration.
+- Hack.md is used primarily for document drafts before being published permanently on GitHub. Both being in markdown, it's a relatively straightforward transition.
+- Google Meet is where most of the internal meetings happen.
+- Discord is used for public meetings which are open to the public.
```
Comment (leohhhn) [inline docs/Discoveries/Gnoland-TPM-audit.md:48]:
```suggestion
- Discord is used for meetings which are open to the public.
```

File: docs/Discoveries/Gnoland-TPM-audit.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+1. When a PR is ready for a review, the whole team is automatically added as reviewers
+2. There's no process on who reviews what. Team members choose randomly (~200 PRs are currently in the repo), or stick to familiar code.
+3. High priority PRs are mentioned in the comm channels, and put on agenda for the [weekly PR review](https://github.com/orgs/gnolang/projects/4/views/1) board.
+4. At least 2 reviews are generally needed, including at least 1 Core team member.
+5. If an issue needs Jae's approval, he is notified via comm channels, as well as on the weekly PR review.
+6. There is no special case handling for external contributors.
+
+### Documentation 
+
+- [docs.gno.land](https://docs.gno.land/) is the public technical documentation repository that's regularly maintained.
```
Comment (leohhhn) [inline docs/Discoveries/Gnoland-TPM-audit.md:83]: The docs are hosted on docs.gno.land - but there are currently two repos for holding the actual docs files:
1. Monorepo - holding content in the `docs/` folder
2. docs.gno.land repo - holding misc files.

**Review by leohhhn — APPROVED** [review]
Nice. Thank you for the analysis. I find basically everything you've found relatable and agree with it. Left 2 minor comments.

File: docs/Discoveries/Gnoland-TPM-audit.md (hunk 2 above)
Comment (Kouteki) [reply docs/Discoveries/Gnoland-TPM-audit.md:83]: Thanks for the clarification

**Review by zivkovicmilos — APPROVED** [review]

**Comment by zivkovicmilos** [comment]
@Kouteki 
Can you please check the spelling CI? 🙏 

And the missing pages for the `build`

**Pull request merged**